scene/gui/video_stream_player.cpp

```diff
@@ -160,6 +160,9 @@ void VideoStreamPlayer::_notification(int p_notification) {
 
 			if (!playback->is_playing()) {
 				emit_signal(SceneStringNames::get_singleton()->finished);
```

Comment:
Thank you for the tip!

I was thinking of not emitting the signal, but had decided to emit it because it is still meaningful in some way (e.g. counting loops/views of the video), which otherwise is only possible for videos looped this way via some hacks in `_process`. But, as you said, it might make sense to wait until someone requests this use-case.

I was not aware of the convention in the AudioStreamPlayer. I would like to adhere to the established semantics of the `finished` signal so I will change it to not emit the signal anymore.